site/content/en/docs/tutorials/multi_node.md

```diff
@@ -42,6 +43,21 @@ multinode-demo       Ready    master   9m58s   v1.18.0
 multinode-demo-m02   Ready    <none>   9m5s    v1.18.0
 ```
 
+```
```

Comment:
Add a note here:

- You can also check the status of your nodes:

Reply:
Added NOTE.